core/testutils/src/main/java/com/mongodb/stitch/core/testutils/sync/SyncIntTestProxy.kt

```diff
@@ -1686,6 +1687,103 @@ class SyncIntTestProxy(private val syncTestRunner: SyncIntTestRunner) {
         assertNoVersionFieldsInLocalColl(syncTestRunner.syncMethods())
     }
 
+    @Test
+    fun testMultiUserSupport() {
```

Comment:
[task] the comments in this test seem out of date. can you update them to reflect what's actually happening?

Reply:
Sure.